### PR TITLE
💬  Make copy for Pixi less aggressive.

### LINKED ISSUE #4696: Pixi: Soften language if a page passes px and we have recommendations
Screen Shot 2020-09-24 at 5 20 02 PM

 - [ ] Change `Fix it now` to `Make improvements`
 - [ ] Remove `but we think you can do better.` as it is mentioned in description. Sounds passive aggressive.

### PR DESCRIPTION
**Pull request by matthiasrohmer**
Fixes #4696.

### CODE REVIEW

**Review by caroqliu — COMMENTED** [review]
@nainar correct me if I'm wrong but I believe `Fix it now` is desired in all other cases, but should be replaced with `Make improvements` when the status returns `passed-with-info`?

**Comment by nainar** [comment]
Yes to what @caroqliu said

File: pages/content/pixi/index@de.md [hunk 1]
```diff
@@ -57,6 +56,8 @@ scriptText:
     fileAnIssue: Melde ein Problem bei AMP
     recommendation: Empfehlung
     recommendations: Empfehlungen
+  buttonFixIt: Jetzt verbessern
```
Comment (caroqliu) [inline pages/content/pixi/index@de.md:59]:
```suggestion
  buttonFixIt: Jetzt beheben
```

**Review by caroqliu — APPROVED** [review]
Looks great!
image

**Pull request merged**